docs/admin/release_notes/version_3.4.md

```diff
@@ -0,0 +1,26 @@
+# v3.4 Release Notes
+
+This document describes all new features and changes in the release. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
```

Comment:
Does it make sense to add a note here about the app being in maintenance-only mode at this point?